jenkins-design-language/src/js/components/status/StatusIndicator.jsx

```diff
@@ -6,15 +6,17 @@ import SvgStatus from './SvgStatus';
 
 const validResultValues = {
     // Mostly from io/jenkins/blueocean/rest/model/BlueRun.java
+    // TODO: Do we want to move this? I have NFI
```

Comment:
It would be great if this was enum defined in blue ocean somewhere, but we don't have a great way of sharing code from there down to the JDL. I think we should leave this TODO and revisit after XP 2.0 lands.